glaciers/glaciers.ipynb

```diff
@@ -15,17 +15,7 @@
     "\n",
     "<img align='right' style='padding:0px 0px 40px 30px' src='./assets/screenshot.png' width=\"40%\"/>\n",
     "\n",
-    "This notebook provides an annotated [HoloViews](https://holoviews.org)+[Panel](https://panel.holoviz.org) implementation of a [dashboard originally developed by Fabien Maussion in Plotly+Dash](https://github.com/OGGM/OGGM-Dash/blob/master/apps/explore/app.py) for viewing data about the Earth's glaciers from the [Open Global Glacier Model](https://oggm.org).  To run it, first:\n",
-    "\n",
-    "    conda install -c pyviz pandas=0.24 param=1.10.0 panel=0.10.1 holoviews=1.13.5 datashader=0.11.1\n",
-    "\n",
-    "Next, save the [data file](https://cluster.klima.uni-bremen.de/~fmaussion/misc/oggm_glacier_explorer.csv) as `data/oggm_glacier_explorer.csv` (and gzip it if desired).\n",
-    "\n",
-    "The dashboard can then be used here as a cell in the Jupyter notebook, or you can run it as a separate server using:\n",
-    "\n",
-    "    panel serve glaciers.ipynb --show\n",
-    "    \n",
-    "This notebook is essentially the same as [Glaciers.ipynb](https://anaconda.org/jbednar/glaciers) but uses unaggregated data that is practical only with [Datashader](https://datashader.org)."
+    "This notebook provides an annotated [HoloViews](https://holoviews.org)+[Panel](https://panel.holoviz.org) implementation of a [dashboard originally developed by Fabien Maussion in Plotly+Dash](https://github.com/OGGM/OGGM-Dash/blob/master/apps/explore/app.py) for viewing data about the Earth's glaciers from the [Open Global Glacier Model](https://oggm.org)."
```

Comment:
This should link to hvPlot, not HoloViews, now that the notebook depends mostly on hvPlot.